pkg/sql/coltypes/aliases.go

```diff
@@ -21,6 +21,7 @@ var (
 	Bool = &TBool{Name: "BOOL"}
 	// Boolean is an immutable T instance.
 	Boolean = &TBool{Name: "BOOLEAN"}
+
```

Comment:
![space](https://space-facts.com/wp-content/uploads/magellanic-clouds.png)

Reply:
Ah, I just noticed this was some surgical spacing. Carry on :)

Reply:
Intentional 
![space-1](https://user-images.githubusercontent.com/5438456/34117590-2099278a-e3ea-11e7-9e61-5802ccb04802.jpg)